helper/restore_helper.go

```diff
@@ -33,41 +35,49 @@ const (
 )
 
 var (
-	contentRE *regexp.Regexp
+	Restorer IRestoreHelper = new(RestoreHelper)
+
```

Comment:
```suggestion
```

Reply:
Restorer is used [in the helper.gp](https://github.com/arenadata/gpbackup/blob/247d924cd49029eff328e004aa54353c9635c570/helper/helper.go#L260)

Reply:
https://github.com/arenadata/gpbackup/pull/102#discussion_r1792846688

Reply:
it used there for the purpose